KsApi/models/DiscoveryParams.swift

```diff
@@ -147,3 +148,30 @@ private func stringIntToBool(_ string: String?) -> Decoded<Bool?> {
     .map { .success($0 == 0 ? nil : $0 == 1) }
     .coalesceWith(.failure(.custom("Could not parse string into bool.")))
 }
+
+private func decodeToGraphCategory(_ json: JSON?) -> Decoded<RootCategoriesEnvelope.Category> {
+
+  guard let jsonObj = json else {
+    return .success(RootCategoriesEnvelope.Category(id: "-1", name: "Unknown Category"))
```

Comment:
Not sure I understand the `-1` bit?

Reply:
-1 is the id of a unrecognized category. a need to pass a value to the `RootCategory` class initializer, so it can be uesd on the expandable/selectable cells configuration and `DiscoveryNavigationHeaderViewController`